wrappers/map.js

```diff
@@ -52,8 +54,17 @@ class Map extends EventEmitter {
       this.geojsonLayers[name].removeFrom(this.map)
       delete this.geojsonLayers[name]
     }
+    
+    // if clustering is defined then add a marker cluster layer
+    // else add a geoJson layer
+    if (typeof config.cluster !== 'undefined' && config.cluster) {
+      layer = new L.MarkerClusterGroup(config.cluster);
```

Comment:
Seems to work as expected with true, false, undefined, null and {}

I'm not getting icons for the clusters though so looking into that now